.test-infra/jenkins/job_PostCommit_Java_ValidatesRunner_PortabilityApi_ExecutableStage_Dataflow.groovy

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import PostcommitJobBuilder
+
+
+// This job runs the suite of ValidatesRunner tests against the Dataflow
+// runner.
+PostcommitJobBuilder.postCommitJob('beam_PostCommit_Java_ValidatesRunner_PortabilityApi_ExecutableStage Dataflow_Gradle',
```

Comment:
I agree, with the small caveat that I would avoid the "VR" acronym, since it sounds like it means "virtual reality" to anyone who doesn't already know about ValidatesRunner tests.

Reply:
We should remove the _Gradle suffix from the job names as well. This is an artifact from when we were migrating from Maven -> Gradle and had both job types.

I'll go through and remove _Gradle from existing jobs; please remove from this job to be consistent.